FF113 RTCPeerConnectionStats - add docs (#26661)
mdn/content: files/en-us/web/api/rtcpeerconnectionstats/id/index.md

---
title: "RTCPeerConnectionStats: id property"
short-title: id
slug: Web/API/RTCPeerConnectionStats/id
page-type: web-api-instance-property
browser-compat: api.RTCPeerConnectionStats.id
---

{{APIRef("WebRTC")}}

> The **`id`** property of the {{domxref("RTCPeerConnectionStats")}} dictionary is a string which uniquely identifies the object
> for which this `RTCStats` object provides statistics.
> 
> Using the `id`, you can correlate two or more `RTCStats`-based objects in order to monitor statistics over time for a given WebRTC object, such as an {{Glossary("RTP")}} stream, an {{domxref("RTCPeerConnection")}}, or an {{domxref("RTCDataChannel")}}.
> 
> ## Value
> 
> A string which uniquely identifies the object for which this `RTCStats`-based object provides statistics.

Author: ```suggestion
The **`id`** property of the {{domxref("RTCPeerConnectionStats")}} dictionary is a string which uniquely identifies the object used to populate these statistics.

Using the `id`, you can correlate this statics object with others, in order to monitor statistics over time for a given WebRTC object, such as an {{domxref("RTCPeerConnection")}}, or an {{domxref("RTCDataChannel")}}.

## Value

A string which uniquely identifies the object for which this `RTCPeerConnectionStats` object provides statistics.
```
